src/sdk/account/toNexusAccount.ts

```diff
@@ -178,16 +180,16 @@ export const toNexusAccount = async (
     args: [signerAddress, index, [], 0]
   })
 
-  const defaultedActiveModule =
-    activeModule ??
+  let defaultedActiveModule =
+    activeValidationModule ??
```

Comment:
we can also call this activeValidator
but can keep this as note for now and still merge PR.